habitat/core/dataset.py

```diff
@@ -89,6 +90,16 @@ def get_episodes(self, indexes: List[int]) -> List[T]:
         """
         return [self.episodes[episode_id] for episode_id in indexes]
 
+    def get_episode_iterator(self):
```

Comment:
Maybe name it `get_cycle_episode_iterator` and add more method later with more sophisticated functionality.

Reply:
Actually, I plan to change `episode_iterator` a class property, and read functionality options from config file (added configs for this). Since a new PR will happen very soon, I think I'll leave this as is for now if you agree @mathfac .